### PR TITLE
Select Statement to retrieve the modified date of all the items in product table formatted in Date Format.

### PR DESCRIPTION
**Pull request by Harshithj88**

### CODE REVIEW

File: SqlIntro/ProductRepository.cs [hunk 1]
```diff
@@ -25,12 +25,21 @@ public IEnumerable<Product> GetProducts()
...
                 var cmd = conn.CreateCommand();
-                cmd.CommandText = ""; //TODO:  Write a SELECT statement that gets all products
+                cmd.CommandText = "select ModifiedDate from product";
                 var dr = cmd.ExecuteReader();
                 while (dr.Read())
                 {
-                    yield return new Product { Name = dr["Name"].ToString() };
+                    yield return new Product
+                    {
+                        //ProductID = (int)dr["ProductID"],
```
Comment (soitgoes) [inline SqlIntro/ProductRepository.cs]: These can be added back if your select statement has them.

**Comment by Harshithj88** [comment]
Successfully Executed Insert, Update and Delete Statements.

File: SqlIntro/DapperDB.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+using Dapper;
+using MySql.Data.MySqlClient;
+
+namespace SqlIntro
+{
+    public class DapperDb
+    {
+        private readonly string _connectionString;
+
+        public DapperDb(string connectionString)
```
Comment (soitgoes) [inline SqlIntro/DapperDB.cs]: Would prefer "ProductRepository" as a class name.  You can alter the namespace to be SqlIntro.Dapper if you like.

Comment (soitgoes) [reply SqlIntro/DapperDB.cs]: I would love to see this reworked to use a IDbConnection into the constructor

**Pull request closed without merging**